Rules/Strings.resx

```diff
@@ -726,4 +726,13 @@
   <data name="ProvideDefaultParameterValueName" xml:space="preserve">
     <value>ProvideDefaultParameterValue</value>
   </data>
+  <data name="AvoidUsingDeprecatedManifestFieldsCommonName" xml:space="preserve">
+    <value>Avoid Using Deprecated Manifest Fields</value>
+  </data>
+  <data name="AvoidUsingDeprecatedManifestFieldsDescription" xml:space="preserve">
+    <value>Some manifest fields are obsolte in the latest PowerShell version. Please update with the latest manifest fields in modules to avoid PowerShell version inconsistency.</value>
```

Comment:
No. One way to get them is to enable PS 2.0 on a machine (windows feature), generate a module manifest and examine this using Test-MM from 5.0 version.